packages/data-transport-layer/src/services/main/service.ts

```diff
@@ -54,8 +54,13 @@ export class L1DataTransportService extends BaseService<L1DataTransportServiceOp
   } = {} as any
 
   protected async _init(): Promise<void> {
+    this.logger.info('Initializing L1 Data Transport Service...')
+
     this.state.db = level(this.options.dbPath)
     await this.state.db.open()
+    this.logger.info('Opened db', {
```

Comment:
Why did you decide to add this?

Reply:
originally i went through all of our async calls and logged around that so we know if any of them fails - is a db malfunction easy enough for spot for so we don't need these?

Reply:
It is definitely too low-level for `INFO`. It is probably too low-level even for `DEBUG`. Maybe `TRACE`?

Reply:
doesn't seem like this would be a hard to find error so i'm good leaving it out!